src/core/source_buffers/segment_inventory.ts

```diff
@@ -777,7 +814,84 @@ function guessBufferedEndFromRangeEnd(
 
   } else {
     log.debug("SI: Segment appears immediately garbage collected at the end",
-              lastSegmentInRange.bufferedEnd, rangeEnd);
+              bufferType, lastSegmentInRange.bufferedEnd, rangeEnd);
     lastSegmentInRange.bufferedEnd = rangeEnd;
   }
 }
+
+/**
+ * Pretty print the inventory, to easily note which segments are where in the
+ * current buffer.
+ *
+ * This is mostly useful when logging.
```

Comment:
Why not adding an example here ?

Reply:
I was too lazy!
But you're right, I should do that.

Reply:
Done